signer/src/network/libp2p/network.rs

```diff
@@ -100,6 +101,10 @@ impl MessageTransfer for P2PNetwork {
             }
         }
     }
+
+    fn receiver_stream(&self) -> BroadcastStream<SignerSignal> {
```

Comment:
> I think we also could also get rid of `signal_rx` as an instance field and `subscribe()` locally in the methods that use it.

Oh I thought it was there so that we always have a live receiver.

Reply:
Did the filtering in https://github.com/stacks-network/sbtc/pull/870/commits/b4dafff3934c9fcecbe277727991132c2254fdef

Reply:
> Oh I thought it was there so that we always have a live receiver.

Ah yeah maybe it was :+1: